api/v1/highlights/PKPHighlightsHandler.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+/**
+ * @file api/v1/highlights/PKPHighlightsHandler.php
```

Comment:
Now that we've got namespacing, I'd prefer not to add new `PKP`-prefixed classes. Let the namespaces speak for themselves, IMO.